contrib/implements/internal/implements/cast.go

```diff
@@ -64,35 +67,40 @@ func stubCFunctions(libname string) (CFunctions, error) {
 	if cstub == "" {
 		return nil, fmt.Errorf("no C stub available for '%s'", libname)
 	}
-	var conf cc.Config
-	conf.PreprocessOnly = true
-	conf.IgnoreInclude = regexp.MustCompile(`^<.+>$`)
+	conf, err := cc.NewConfig(runtime.GOOS, runtime.GOARCH, "-D_FILE_OFFSET_BITS=64")
+	if err != nil {
+		return nil, err
+	}
+
 	src := []cc.Source{
-		{Name: "typestubs", Value: typeStubs, DoNotCache: true},
-		{Name: libname, Value: cstub, DoNotCache: true},
+		{Name: "<predefined>", Value: conf.Predefined},
+		{Name: "<builtin>", Value: cc.Builtin},
+		{Name: "typestubs", Value: typeStubs},
+		{Name: libname, Value: cstub},
 	}
-	inc := []string{"@", "/usr/local/include", "/usr/include"}
-	cAST, err := cc.Parse(&conf, inc, nil, src)
+	cAST, err := cc.Parse(conf, src)
 	if err != nil {
 		return nil, err
 	}
-	cfMap := map[cc.StringID]CFunction{}
-	deprecated := cc.String("deprecated")
-	for i := range cAST.Scope {
-		for _, n := range cAST.Scope[i] {
+	cfMap := map[cc.StringValue]CFunction{}
+	for i := range cAST.Scope.Nodes {
+		for _, n := range cAST.Scope.Nodes[i] {
 			if n, ok := n.(*cc.Declarator); ok &&
-				!n.IsTypedefName &&
+				!n.IsTypename() &&
 				n.DirectDeclarator != nil &&
 				(n.DirectDeclarator.Case == cc.DirectDeclaratorFuncParam ||
 					n.DirectDeclarator.Case == cc.DirectDeclaratorFuncIdent) {
 				name := n.Name()
-				if _, exists := cfMap[name]; !exists {
-					_, isDeprecated := n.AttributeSpecifierList.Has(deprecated)
+				if _, exists := cfMap[cc.StringValue(name)]; !exists {
+					isDeprecated := false
+					if strings.Contains(n.DirectDeclarator.IdentifierList.String(), "deprecated") {
```

Comment:
I see that `AttributeSpecifierList` is no longer part of `DirectDeclarator` but this changed detection logic doesn't seem to work. I couldn't see functions marked as deprecated in _implements.txt_.